docs/getting-started/switch-to-composer.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Switch to Composer"
+description: "How to switch your plugin based Timber theme over to the Composer based version."
+menu:
+  main:
+    parent: "getting-started"
+---
+
+With the upcoming release of Timber 2, we will stop releasing Timber as a plugin. We advise everyone to switch to the Composer based install of Timber 1 as soon as possible as a first step towards upgrading to Timber 2.
+
+## Recommended upgrade path
+
+1. Get a local development version of your website up and running.
+2. Check if you are running the latest version of the Timber plugin. If not, update it and check if your website still runs as expected.
+
+4. Install the latest 1.x version of Timber via Composer
+5. Load the Composer autoloader and initialize Timber
+5. Load the composer autoloader and initialize Timber 
+6. Check if your website still runs as expected
+7. Deploy your changes to your live website
+
+We highly recommend doing these steps on a local development version of your website. If you don’t have one yet, you could use [Local by Flywheel](https://localbyflywheel.com/) to get one up and running in a few minutes.
+
+#### 1.1 How to install Composer
+If you don’t have Composer installed yet, you can follow the [official installation guide](https://getcomposer.org/doc/00-intro.md).
+
+
+### 2. Check if you are running the latest version of the Timber plugin
+We want to make sure that you are running the latest version of the Timber plugin before we start the upgrade process. This way we can be sure that any issues that might occur are not caused by an outdated version of the plugin.
+
+### 3. Disable the Timber plugin
+Once you are sure that you are running the latest version of the Timber plugin, you can disable it. This will make sure that the plugin is not loaded anymore and does not interfere with the Composer based version of Timber.
+
```

Comment:
```suggestion

Please note that your website will at this point in time no more work as there is no Timber available. This will be fixed in the next step.
```

Reply:
Good addition, I changed it a bit to:

`Please note that you website will throw erros at this point, as there is no Timber available. This will be fixed in the next two steps.`